spark/wordcount/README.md

```diff
@@ -0,0 +1,102 @@
+# Dataproc Bigtable WordCount Sample
+
+## Overview
+
+This sample demonstrates how to use  [Google Cloud Dataproc](https://cloud.google.com/dataproc), which
+provides a managed [Apache Spark](https://spark.apache.org/) environment with
+[Google Cloud Bigtable](https://cloud.google.com/bigtable/docs).
+
+This specific example provides the classic map reduce example of counting words
+in a text file.
+
+## Prerequisites
+
+1. A [Google Cloud project](https://console.cloud.google.com/) with billing enabled. Please
+be aware of [Bigtable](https://cloud.google.com/bigtable/pricing)
+and [Dataproc](https://cloud.google.com/dataproc/docs/resources/pricing) pricing.
+
+1. You have the [Google Cloud SDK](https://cloud.google.com/sdk/) installed.
+
+1. You have [Scala](https://www.scala-lang.org/) installed.
```

Comment:
Local Scala installations are never necessary with Spark (though they can help for debugging).